ietf/templates/meeting/attendance.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+{# Copyright The IETF Trust 2024, All Rights Reserved #}
+{% load origin %}
+{% block title %}Bluesheet for {{session}}{% endblock %}
+{% block content %}
+    {% origin %}
+    <h1>
+        Attendance for {{session}}
+    </h1>
+    <div class="alert alert-info my-3">
```

Comment:
We should put something like this here:
"If the affiliation listed here needs to be updated, request the change using support@ietf.org. Note which sessions you are wanting to change in your request."